### ISSUE TITLE
License Plate from other area

### ISSUE THREAD

**Issue by rubayetfaisal** [nomanvb/Number_Plate_CSharp#1]
i found that it perfectly works for Dhaka metro area but what about other areas like Chittagong ,Khulna and others area.

**Comment by nomanvb** [comment]
Yes, we have not train our model for some other area of Bangladesh. I will request community to upload other area number plates. we don't need images for Dhaka and Dhaka Metro.
